docs/src/content/docs/guides/sidebar.mdx

```diff
@@ -138,7 +138,12 @@ The configuration above generates the following sidebar:
 
 Starlight can automatically generate a group in your sidebar based on a directory of your docs.
 This is helpful when you do not want to manually enter each sidebar item in a group.
-Pages will be sorted alphabetically by filename by default.
+
+By default, pages will be sorted in alphabetical order according to the filenames in your [`defaultLocale`](/reference/configuration/#defaultlocale). 
```

Comment:
If possible to hold the docs update. The sorting may change, again (by bad 😅 ).

For context #1151, we might use the slug instead of the filename.
```suggestion
By default, pages will be sorted in alphabetical order according to the [`slug`](reference/overrides/#slug) in your [`defaultLocale`](/reference/configuration/#defaultlocale). 
```

Reply:
#1151 merged 💜

Reply:
Wonderful